src/lib/y2users/autoinst/reader.rb

```diff
@@ -81,6 +87,11 @@ def read
       # @return [AutoinstProfile::LoginSettingsSections]
       attr_reader :login_settings_section
 
+      # Profile section describing the default configuration for new users
+      #
+      # @return [AutoinstProfile::UserDefaultsSections]
```

Comment:
Typo: UserDefaultsSection~~s~~

Reply:
Correct. I just realized I made the typo by copy-pasting from the line above (LoginSettingsSections)